modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -1146,6 +1146,7 @@ TEST_P(Test_ONNX_layers, Split)
         applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_NN_BUILDER);
     if (backend == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH)
         applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_NGRAPH);
+    testONNXModels("split_0");
```

Comment:
This test does not cover code changes. It goes not to `if (layerParams.has("split"))`  branch but `else if (node_proto.input_size() == 2)`.